projects/plugins/jetpack/class.json-api.php

```diff
@@ -175,7 +175,7 @@ class WPCOM_JSON_API {
 	 */
 	public static function init( $method = null, $url = null, $post_body = null ) {
 		if ( ! self::$self ) {
-			$class      = function_exists( 'get_called_class' ) ? get_called_class() : __CLASS__; // phpcs:ignore PHPCompatibility.PHP.NewFunctions.get_called_classFound
+			$class      = function_exists( 'get_called_class' ) ? static::class : __CLASS__; // phpcs:ignore PHPCompatibility.PHP.NewFunctions.get_called_classFound
 			self::$self = new $class( $method, $url, $post_body );
```

Comment:
This replacement results in code that really doesn't make any sense.

But since we dropped PHP 5.2 support years ago, we can just do it like this.
```suggestion
			self::$self = new static( $method, $url, $post_body );
```